src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const people = [...document.querySelectorAll('tbody tr')];
+
+function toNumber(salary) {
```

Comment:
Get used to name functions according to these rules:
![image](https://user-images.githubusercontent.com/96409563/232069656-16cef5fc-9223-4137-8bd8-ab532d9f7ad8.png)